tsl/test/isolation/specs/dist_restore_point.spec.in

```diff
@@ -63,8 +64,8 @@ step "s2_create_dist_rp" { SELECT restore_point > pg_lsn('0/0') as valid_lsn FRO
 step "s2_insert"         { INSERT INTO disttable VALUES ('2019-08-02 10:45', 0, 0.0); }
 step "s2_begin"          { BEGIN; }
 step "s2_commit"         { COMMIT; }
-step "s2_add_dn"         { SELECT node_name FROM add_data_node('data_node_4', host => 'localhost', database => 'cdrp_4'); }
```

Comment:
are there data_node_n changes related?

Reply:
Yes, I've mentioned about it in the message above. It must create a new data node, but it was already created in the previous test